### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by tm1000** [bitfocus/companion-module-bss-soundweb#2] [Ideas]
The documentation you provided so far is very good but I think it would be or might be handy to have some pictures. I only utilize the outdated London Architect but I could provide details on how to use that with DI, eg through a PR.

I honestly can't think of anything else you need to add to this module (at this time). The feedbacks work as expected and the buttons work too (from triggering a GXP logic gate to mute buttons it all works).

Seems like lots of folks are moving to q-sys these days but the London gear is very rock solid (knock on wood)

**Comment by AndyFug** [comment]
Hi.  Yes.  I agree.  To be perfectly honest, when I was writing the Help I wasn't too sure how to add images without them being hosted elsewhere on the web... I will get my head around this and sort it out. :). Thank you.

**Reply by tm1000** [reply]
You can put the images in the repo I've seen other projects do it.

I am not so clear on how bitfocus publishes these modules into their source. I suppose the images would bloat that unfortunately.

**Reply by AndyFug** [reply]
Yes.  I've found another example of a module using a local image in the help.md file, so all good.

I think the key screen captures we need are how to find the parameter/object addresses in LA & AA and then how that translates into the fully qualified address format in the module.  I'll see if I can do it as a short (video) screen capture for each application, because that will probably be the most succinct way to communicate it.
